### ISSUE TITLE
[BUG] Change page name in wizard type.

### ISSUE THREAD

**Issue by homashevchenko** [formio/formio.js#2546]
- Hosting type
  - [ ] Form.io
  - [x] Local deployment
- Formio.js version: 4.8.1 || 4.9.0-rc.10 
- Browser: Chrome
- Browser version: 80.0.3987.132

## Steps to Reproduce
1. Create wizard form
2. Change page label
3. Click on different tabs, or save and reload form 

## Expected behavior
Page label is changed (In panel component and page tab)
## Observed behavior
Page label not changed (In panel component and page tab)
## Example
Replicated on versions: 4.8.1, 4.9.0-rc.10;
On "https://formio.github.io/ngFormBuilder/" looks fine.
![image](https://user-images.githubusercontent.com/22855299/76961106-71130600-6925-11ea-957f-8b7cf4edd3b9.png)
Thanks

**Comment by wag110894** [comment]
@homashevchenko, 

We are unable to replicate this on our side. Can you confirm that you saved the form? Here is a screenshot of this working: https://share.getcloudapp.com/DOu8k7X8

**Comment by homashevchenko** [comment]
The reason is because:
"saveComponent" function font change panel component in "form" object.
![image](https://user-images.githubusercontent.com/22855299/77052339-5ac48300-69d5-11ea-8000-47f7ab4065f5.png)

**Comment by homashevchenko** [comment]
if add "_this9.form.components[index] = submissionData;" line. all works fine
![image](https://user-images.githubusercontent.com/22855299/77064901-01fee580-69e9-11ea-9e03-45cddbcc25c2.png)

**Comment by travist** [comment]
We are unable to reproduce this on our end. Are you using https://next.form.io to test this? Do you have a hosted application we can take a look at?

**Comment by gregbown** [comment]
The only thing that resolves this is calling the getter builder.instance.schema in your on change event handler for your builder.  In my [demo](https://stackblitz.com/edit/formio-builder-rncrmy?file=index.js), If I swap out "builder.instance.schema" with "builder.instance.form" the panel update breaks in the manner described in this bug.  I posted a similar bug [here](https://github.com/formio/formio.js/issues/2389).

Keep in mind, although it verifies the issue, the demo is pulling latest which is currently 4.9.3 which is also broken with an infinite loop change event.

**Comment by travist** [comment]
@gregbown we believe we resolved the infinite change event issue with latest versions. Please try that version and let us know if that resolves the problem with trying the latest version.

**Comment by gregbown** [comment]
@travist I definitely appreciate all the work put into refactoring events, however; the issue in this bug and mine are being able to edit a panel without it breaking the entire form , secondly, the infinite loop issue is still occurring in this use case.

Building out forms is not possible unless the getter builder.instance.schema is called.

Also, while your fix was quite comprehensive, if  the getter builder.instance.schema is called, the change event **still goes into an infinite loop**, even with all your latest changes.
My fix below resolves the infinite loop when calling the getter builder.instance.schema, however I would like your expert opinion on:
a) Why do I need to call the getter builder.instance.schema for the builder to continue working?
    shouldn't something be done by default internally so the builder keeps working?
b) Is my fix really needed or the best way to resolve the infinite loop when using the getter?

[Fix for infinite loop when calling getter builder.instance.schema](https://github.com/formio/formio.js/blob/master/src/Webform.js#L1260)
```
    if ((!flags || !flags.noEmit) && changed) {
      this.emit('change', value);
      isChangeEventEmitted = true;
    }
```

**Comment by gregbown** [comment]
@travist This is fixed with 4.10.2.  Thank you!
Panel changes no longer break form in wizard mode. Also, calling schema getter is no longer needed and the infinite loop is also fixed.
